ocaml/xenops/xenops.ml

```diff
@@ -228,7 +228,7 @@ let stat ~xs d =
 		| x -> string_of_kind x in
 	let frontend_type = match d.frontend.kind with
 		| Vbd _ | Tap ->
-			let be = frontend_path_of_device ~xs d in
+			let be = frontend_ro_path_of_device ~xs d in
 			(try if xs.Xs.read (sprintf "%s/device-type" be) = "cdrom" then "cdrom" else "disk" with _ -> "??")
```

Comment:
These are the only two changes that do not seem directly reflected in the other PR. I think I understand why we read the frontend_state from the rw path, just wanted to ask why here we read from the ro path instead.